tests/functional/DependencyCheckTest.php

```diff
@@ -115,16 +115,17 @@ public function testProfiler()
 
 	public function testDatabase()
 	{
-		$configCache = $this->dice->create(Cache::class);
-		$configCache->set('database', 'disable_pdo', true);
+		// PDO needs to be disabled for PHP 7.2, see https://jira.mariadb.org/browse/MDEV-24121
+		if (version_compare(PHP_VERSION, '7.3') < 0) {
```

Comment:
If it is just for PHP 7.2, why not compare exactly?

Reply:
This hadn't worked in my tests.

Reply:
AFAIK this problem occurs with every version lower than 7.3 - so this test is okay from my point of view. Of course I could compare against `PHP_MAJOR_VERSION` and `PHP_MINOR_VERSION` but this doesn't look better, I think.

Reply:
PHP 7.2 is the current stable version anyway, we do not need to support lower versions.